Topics/Development_Process/Build_Requirements/CMakeLists_txt.md

```diff
@@ -0,0 +1,199 @@
+# **Learning CMakeLists.txt**
+
+## Table of Contents
+### [1. Brief Introduction to CMake](#brief-introduction-to-cmake)
+### [2. What is CMakeLists.txt?](#what-is-cmakeliststxt)
+### [3. Creating CMakeLists.txt](#creating-cmakeliststxt)
+### [4. Other Notes](#other-notes)
+### [5. References](#references)
+
+---
+## Brief Introduction to CMake
+
+This article will help readers understand what `CMakeLists.txt` is, why it is used, and provide instructions on how to create a `CMakeLists.txt`. Before getting started, it is important to understand what is CMake. Note that it is presumed that the reader should have the very basic knowledge of C++, Make, and MakeFile.
+
+Unlike Make, which is a build system that drives the compiler and other build tools to build your code, CMake (or Cross-platform Make) is a "generator" of build systems that can be used for general purpose build. It is much more high-level that it can generate a low-level build script in Ninja or Make or many other build systems, and then you run it.
+
+The key feature of CMake is that it supports the cross-platform discovery of system libraries, and automatic discovery and configuration of the toolchain, which is much easier to use than Make.
```

Comment:
Wording: "which is" -> "making it" or "which makes it" or something like that, probably

Reply:
I think the original one is good enough, probably

Reply:
Alright, if you say so. The point ultimately gets across either way.